libraries/c_sdk/standard/mqtt/src/iot_mqtt_serialize.c

```diff
@@ -79,76 +79,12 @@
  */
 #define UINT8_CHECK_BIT( x, position )    ( ( x & ( 0x01 << position ) ) == ( 0x01 << position ) )
 
-/*
- * Positions of each flag in the "Connect Flag" field of an MQTT CONNECT
- * packet.
- */
-#define MQTT_CONNECT_FLAG_CLEAN                     ( 1 )  /**< @brief Clean session. */
-#define MQTT_CONNECT_FLAG_WILL                      ( 2 )  /**< @brief Will present. */
-#define MQTT_CONNECT_FLAG_WILL_QOS1                 ( 3 )  /**< @brief Will QoS1. */
-#define MQTT_CONNECT_FLAG_WILL_QOS2                 ( 4 )  /**< @brief Will QoS2. */
-#define MQTT_CONNECT_FLAG_WILL_RETAIN               ( 5 )  /**< @brief Will retain. */
-#define MQTT_CONNECT_FLAG_PASSWORD                  ( 6 )  /**< @brief Password present. */
-#define MQTT_CONNECT_FLAG_USERNAME                  ( 7 )  /**< @brief Username present. */
-
 /*
  * Positions of each flag in the first byte of an MQTT PUBLISH packet's
  * fixed header.
  */
-#define MQTT_PUBLISH_FLAG_RETAIN                    ( 0 )  /**< @brief Message retain flag. */
-#define MQTT_PUBLISH_FLAG_QOS1                      ( 1 )  /**< @brief Publish QoS 1. */
-#define MQTT_PUBLISH_FLAG_QOS2                      ( 2 )  /**< @brief Publish QoS 2. */
-#define MQTT_PUBLISH_FLAG_DUP                       ( 3 )  /**< @brief Duplicate message. */
-
-/**
- * @brief The constant specifying MQTT version 3.1.1. Placed in the CONNECT packet.
- */
-#define MQTT_VERSION_3_1_1                          ( ( uint8_t ) 4U )
-
-/**
- * @brief Per the MQTT 3.1.1 spec, the largest "Remaining Length" of an MQTT
- * packet is this value.
- */
-#define MQTT_MAX_REMAINING_LENGTH                   ( 268435455UL )
-
-/**
- * @brief The maximum possible size of a CONNECT packet.
- *
- * All strings in a CONNECT packet are constrained to 2-byte lengths, giving a
- * maximum length smaller than the max "Remaining Length" constant above.
- */
-#define MQTT_PACKET_CONNECT_MAX_SIZE                ( 327700UL )
-
-/*
- * Constants relating to CONNACK packets, defined by MQTT 3.1.1 spec.
- */
-#define MQTT_PACKET_CONNACK_REMAINING_LENGTH        ( ( uint8_t ) 2 )    /**< @brief A CONNACK packet always has a "Remaining length" of 2. */
-#define MQTT_PACKET_CONNACK_SESSION_PRESENT_MASK    ( ( uint8_t ) 0x01 ) /**< @brief The "Session Present" bit is always the lowest bit. */
+#define MQTT_PUBLISH_FLAG_DUP    ( 3 )                     /**< @brief Duplicate message. */
```

Comment:
Should this file, iot_mqtt_serialize.c completely disappear and instead get replaced by mqtt_light_weight.c ?